src/include/utils/tuplesort_mk_details.h

```diff
@@ -115,7 +115,6 @@ static inline int32 mke_get_run(MKEntry *e)
 }
 static inline void mke_set_run(MKEntry *e, int32 run)
 {
-    Assert(run >= 0 && run <= MKE_MAX_RUN);
```

Comment:
Maybe leave the check greater than zero? And why do we have a limit on the number of runs in general?

Reply:
> Maybe leave the check greater than zero?

done

Reply:
> And why do we have a limit on the number of runs in general?

maybe because of https://github.com/arenadata/gpdb/blob/0cd01d81b559f1479759087833b964529d3b09e1/src/include/utils/tuplesort_mk_details.h#L29